package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "ai-flow-dashboards",
```

Comment:
shouldn't this package name be `opensearch-ai-flow-dashboards`?

Reply:
I'm following the standard naming conventions of OSD plugins, which is `<plugin-name>` + `'dashboards'` - may be camel case or kebab case